src/components/Navbar/Navbar.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react';
-import './NavBar.scss';
+import './Navbar.scss';
```

Comment:
I think we need to fix the capitalization of NavBar on both of our ends. Seems to be inconsistent.